gulpfile.js

```diff
@@ -93,6 +94,21 @@ gulp.task('styles', () => {
         .pipe(gulp.dest(`${serverPublicFolderPath}/css`));
 });
 
+// Transfer vendor css files to the publix folder
+gulp.task('vendorStyles', () => {
+    const processors = [];
+
+    if (!isDev) {
+        processors.push(cssnano());
+    }
+
+    return gulp.src(vendorStylesGlob)
+        .pipe(postcss(processors))
```

Comment:
Just use an array litteral

Reply:
Actually the array literal will not work because we only add some stuff in dev mode